cEP-0020.md

```diff
@@ -0,0 +1,167 @@
+
+## Background
+
+### Newcomer Experience
+
+- How to attract more newcomers
+  - How to cope with the increasing amounts of newcomers
+- How to teach and help newcomers scalable so we can keep working on coala.
+  - Talk to Docs team
+- How to improve newcomer -> dev/maintainer ratio
+- How to increase amount of feedback we get
+  - How to use feedback to improve coala and processes
+
+## Why Gamification?
+
+As coala tries to be welcoming organization for newcomers, giving them clear pathways and as much direct assistance as possible. However, it is already known that newcomers face many barriers while attempting to contribute to the open source community for the first time. Some barriers they face include orientation issues that can potentially demotivate newcomers from placing their first contribution. On the other hand, gamification is widely used to engage and motivate people to accomplish task and improve their performance.Therefore, the goal of this project is to use gamification to orient and motivate newcomers to overcome onboarding barriers to contribute and engage with coala and its community.
+
+## Process involved in the Gamification system
+
+As the goal of this project is to make use of metrics and gamification to motivate and support newcomers to overcome the orientation barriers to contribute to the community.
+To achieve this goal we have selected three game design elements to help newcomers: Levels, Points, and Badges. For each of these elements, we will have a set of rules to describe the operation of the gamified environment. i.e.:
+
+- **Levels** group different tasks, enabling users to unlock specific and contextualize set of actions. Levels will be applied to make newcomers focus on a given step before moving ahead on the contribution process. E.g.: We will divide our newcomer process into three levels and each level will contain three tasks:
+
+  - Level-I:
+    - Joining our gitter channel and the community
+    - Running CI tools to your fork
+    - Running coala-ci on a popular github repo
+  - Level-II:
+    - Getting assigned to a difficulty/newcomer issue
+    - Creating and merging PR for that issue
+    - Reviewing at least one difficulty/newcomer issue
+  - Level-III:
+    - Getting assigned to a difficulty/low issue
+    - Creating and merging a PR for that issue
+    - Reviewing at least one difficulty/low issue
+
+- **Points** will be the most applied element out of three game design elements. Acquiring points means that the actions performed by the users are in accordance with what was expected. Points can be earned by means of performing actions to the different tasks under levels. E.g.: If a user wants to go to next level he will have to collect a certain number of points from doing the tasks under levels. But in case of Level-II and Level-III if a user is not able to acquire the required number of points after performing all the tasks, he will have to review more PRs to collect the required points.
```

Comment:
so the threshold for leveling up will be more that what you can earn by the minimum right? so that more reviews are encouraged